frontend-svelte/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public --no-clear"
+    "start": "sirv public --no-clear --port=3000"
```

Comment:
The default port (5000) is reserved on the latest version of macos, so i had to change this.